include/lsst/afw/image/Calib.h

```diff
@@ -69,6 +70,25 @@ inline double fluxErrFromABMagErr(double magErr, double mag) {
     return std::abs(-0.4 * magErr * fluxFromABMag(mag) * std::log(10.0));
 }
 
+
+/// Compute AB magnitude from flux in Janskys
+template <typename T>
+ndarray::Array<T, 1> abMagFromFlux(ndarray::Array<T const, 1> const& flux);
```

Comment:
As I think about it, I think these functions should be `abMagFromJansky`, to make the units very explicit (especially given RFC-356).

Reply:
That's an API change that would require an RFC. I would support such an RFC, but do not want to lead one, at least not now.

Reply:
Ok, that's fair. I guess we can just leave it until we throw out all of `Calib` for `PhotoCalib`.